contribs/build.gradle

```diff
@@ -19,6 +19,8 @@ dependencies {
 
     implementation "com.amazonaws:aws-java-sdk-s3:${revAwsSdk}"
     implementation "com.amazonaws:aws-java-sdk-sqs:${revAwsSdk}"
+    implementation "software.amazon.msk:aws-msk-iam-auth:1.1.1"
+    implementation "com.amazonaws:aws-java-sdk-core:1.12.80"
```

Comment:
Can you comment on the need for this dependency? Also why is it pinned to `1.12.80`?